src/ui/screens/DemocracyScreen.test.tsx

```diff
@@ -0,0 +1,109 @@
+    await findByText('5 days 5 hrs');
+    await findByText('Proposals');
+    await findByText('#73');
+    await findByText('batch.utility()');
+    await findByText('Balance:');
+    await findByText('0.0100 KSM');
+    await findByText('Proposer:');
+    await findByText('Shiden Network');
+    await findByText('Set the storage for the current parachain head data immediately.');
+    await findByText('Submit proposal');
+    await findAllByText('Polkassembly');
+    await findAllByText('Submit proposal');
+  });
+
+  it('should open the polkassembly webpage when pressed on Polkassembly', async () => {
+    const linkingSpy = jest.spyOn(Linking, 'canOpenURL');
+    const {findAllByText} = render(<DemocracyScreen navigation={navigation} />);
+    fireEvent.press((await findAllByText('Polkassembly'))[0] as ReactTestInstance);
+    expect(linkingSpy).toBeCalledWith('https://polkadot.polkassembly.io/referendum/211');
+  });
+
+  it('should open submit proposal model verify its initial state and press cancel button', async () => {
+    const {findByText, findAllByText, findByTestId} = render(<DemocracyScreen navigation={navigation} />);
+    fireEvent.press(await findByText('Submit proposal'));
+    await findAllByText('Submit proposal');
+    await findAllByText('Select account');
+    await findByText('PreImage Hash');
+    await findByText('Locked Balance');
+    await findByText('Minimum deposit: 100.0000 DOT');
+    await findByText('Cancel');
+    expect(await findByTestId('proposal-button')).toBeDisabled();
+    fireEvent.press(await findByText('Cancel'));
+  });
+
+  it('should open submit proposal model when pressed on submit proposal', async () => {
+    const {findByText, findByTestId, findByPlaceholderText} = render(<DemocracyScreen navigation={navigation} />);
+    fireEvent.press(await findByText('Submit proposal'));
+
+    const proposalButton = await findByTestId('proposal-button');
+    expect(proposalButton).toBeDisabled();
+
+    const selectAccount = await findByTestId('select-account');
+    fireEvent.press(selectAccount);
+
+    const accountItem = await findByText('Test account name');
+    fireEvent.press(accountItem);
+    expect(proposalButton).toBeDisabled();
+
+    fireEvent.changeText(await findByPlaceholderText('Place your Text'), preImage);
+    expect(proposalButton).toBeDisabled();
+
+    fireEvent.changeText(await findByPlaceholderText('Enter amount'), '24');
+    expect(proposalButton).toBeDisabled();
+
+    fireEvent.changeText(await findByPlaceholderText('Enter amount'), '120');
+    expect(proposalButton).toBeEnabled();
+
+    fireEvent.press(proposalButton);
+
+    expect(mockStartTx).toHaveBeenCalledWith({
+      address: 'G7UkJAutjbQyZGRiP8z5bBSBPBJ66JbTKAkFDq3cANwENyX',
+      params: [
+        '0x66cb3ecc23fd2ef262f8f451e4f0e0dfcfd2e276b3438dc800522df4863659db',
+        {
+          length: 2,
+          negative: 0,
+          red: null,
+          words: [26402816, 17881, ,],
```

Comment:
typo?

Reply:
Are you referring to red? if so the attribute is `red`

Reply:
`words: [26402816, 17881, ,],`